Library/Homebrew/config.rb

```diff
@@ -27,7 +27,11 @@ def mkpath
 HOMEBREW_CACHE_FORMULA = HOMEBREW_CACHE+"Formula"
 
 unless defined? HOMEBREW_BREW_FILE
```

Comment:
I think this should be set in `config.rb`. So some external tools can use `require "global"` without loading `brew.rb`. e.g. `brew ruby` and `brew pry` from `homebrew/dev-tools`.

In fact, I think we can remove setting in `brew.rb`. As `require "global"` which will invoke `require "config"` should be sufficient.